app/src/main/java/com/android/shelfLife/viewmodel/overview/OverviewScreenViewModel.kt

```diff
@@ -120,7 +118,9 @@ constructor(
       viewModelScope.launch {
         listFoodItemsRepository.foodItems.collect { foodItems ->
           foodItems.forEach { foodItem ->
-            if (foodItem.expiryDate!! < Timestamp.now() && foodItem.status != FoodStatus.EXPIRED) {
+            if (foodItem.expiryDate != null &&
```

Comment:
# Good practice, but necessary?
good addition of the null check. I think however that its unnecessary since expiryDate is ALWAYS defined for fooditems.